tasks/main.yml

```diff
@@ -113,6 +110,13 @@
   notify: verify config and restart httpd
   when: _use_tls
 
+- name: remove obsolete files
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - "{{ _vhost_confdir }}/python.conf"
+
```

Comment:
My bad.

How can you possibly define $somedate to something meaningful in advance?